tests/target-browser/target_browser_test.go

```diff
@@ -643,6 +676,18 @@ var _ = Describe("Target Browser Tests", func() {
 				Expect(err).Should(HaveOccurred())
 				Expect(string(output)).Should(ContainSubstring(fmt.Sprintf("flag needs an argument: %s", flagBackupStatus)))
 			})
+			It(fmt.Sprintf("Should get zero backup using flag %s with 'invalid' value", cmd.BackupStatusFlag), func() {
+				args := []string{cmdGet, cmdBackup, flagBackupStatus, "invalid"}
+				backupData := runCmdBackup(args)
+				Expect(len(backupData)).To(Equal(0))
+			})
+
+			It(fmt.Sprintf("Should get one page backup using flag %s with zero value", cmd.BackupStatusFlag), func() {
+				args := []string{cmdGet, cmdBackup, flagBackupStatus, ""}
```

Comment:
what is BackupStatus value being passed in request here if flag is given empty value?

Reply:
backupStatus value is `""`. This will bypass i.e This is not a required field and API will return the default one-page result